### ISSUE TITLE
psc-publish fails on purescript-these

### ISSUE THREAD

**Issue by tfausak** [purescript/purescript#1550]
The description for [purescript-these](https://github.com/purescript-contrib/purescript-these) includes some Unicode characters: https://github.com/purescript-contrib/purescript-these/blob/v0.3.0/bower.json

```
Error in JSON in the output of `bower list --json --offline`:              
  The input could not be parsed as JSON aeson said: Failed reading: Cannot 
  decode byte '\xb1': Data.Text.Internal.Encoding.decodeUtf8: Invalid UTF-8
  stream                                                                   
```

**Comment by paf31** [comment]
Is this an `aeson` thing or an encoding issue?

**Comment by tfausak** [comment]
It's probably an encoding issue. I think `aeson` is doing the right thing, since `\xb1` isn't a valid UTF-8 byte. I opened a pull request against purescript-these to remove the weird characters: purescript-contrib/purescript-these#7.

**Comment by paf31** [comment]
Fixed by purescript-contrib/purescript-these#7?

**Comment by tfausak** [comment]
I'm not at my computer, but I think so. I'll reopen if it persists.

**Comment by garyb** [comment]
Yeah, I think this was an issue with that more than `psc-publish`.

**Comment by tfausak** [comment]
Actually that pull request did not fix this problem. I'm still getting the same error. In fact, I get the error even if I change the description in `bower_components/purescript-these/.bower.json` to `"\u03b1"`. If I had to venture a guess, I'd say that Bower isn't outputting UTF-8.

I kept playing with the description and found that `"a ∨ b ∨ (a ∧ b)"` works. So the "logical or" and "logical and" glyphs don't cause problems, but the "alpha" and "beta" glyphs do.
